src/Data/Masa.Contrib.Data.UoW.EF/README.md

```diff
@@ -7,11 +7,11 @@ Example：
 ```C#
 Install-Package Masa.Contrib.Data.UoW.EF
 Install-Package Masa.Contrib.Data.Contracts.EF
```

Comment:
Whether references to Contracts can be removed

Reply:
Yes it can be deleted, I missed it here, processed